src/css/components/_text-track.scss

```diff
@@ -46,3 +46,11 @@ video::-webkit-media-text-track-display {
   text-align: center !important;
   width: 80% !important;
 }
+
+.video-js .vjs-text-track-display > div {
```

Comment:
What does the `div` here represent? Do we want this CSS set always? Or maybe we could add a class name to this `div` in the scenario we want so we know it is SMART TV related..

I just want to be sure this does not have any unexpected side effects..

Reply:
It is a div that is dynamically generated without a classname it is the parent of the vjs-text-track-cue, all of this styles are just backup styles, are all overwritten by inline styles except in old chrome devices.

![Screenshot 2024-08-21 at 13 35 50](https://github.com/user-attachments/assets/ec0ace88-2946-494a-acbb-fd321d9f930d)

Reply:
But I updated the PR to add those styles only when required, can this be re-reviewed?

Reply:
Cool, I would maybe make that name more descriptive "vjs-text-track-display-inset" or something like that. Otherwise, nice work!